docs/product/components/labels.html

```diff
@@ -93,7 +93,7 @@
 
 <section class="stacks-section">
     {% header "h2", "Status" %}
-    <p class="stacks-copy">When you need to flag labels as required or optional, use the following flags. Use the full word “Required” or “Optional” for these flags instead of using asterisks. If a majority of a form’s inputs are required, it isn’t necessary to show that they’re required until after the form has been submitted with errors. It may be more appropriate to only mark the non-required fields as optional.</p>
+    <p class="stacks-copy">When you need to flag labels as required or optional, use the following flags. Use the full word “Required” or “Optional” for these flags instead of using asterisks. If a majority of a form’s inputs are required, use the asterisk symbol to indicate which fields are required and include a legend. See <a href="/product/components/inputs/#accessibility">Input Accessibility</a> for more details.</p>
```

Comment:
I noticed the [Jira ticket](https://stackoverflow.atlassian.net/browse/STACKS-582) says:

> _Alternate approach_
> Add a required text next to the input label (not recommended unless the preferred approach is not possible)

This seems to slightly contradict the guidance added to the docs since the Jira ticket suggests to avoid the longhand "Required" label unless the `*` approach is not possible. 

@giamir what you added here seems more appropriate IMO, since it avoids the need for a legend when there are few required inputs. I wanted to call it out though to make sure we reconcile the discrepancy and so we don't ignore a reasoning for one approach over another.

---

@CGuindon any strong opinion on the recommendation on when to use `Required` vs `*`?

Reply:
Yup, that's older, I like what @giamir has here.